src/main/java/com/cincialert/cincialert/dto/TrafficIncident.java

```diff
@@ -0,0 +1,28 @@
+package com.cincialert.cincialert.dto;
+
+import lombok.Data;
+
+@Data
+public class TrafficIncident {
+
+    /**
+     * The incident's ID.
+     */
+    private String incidentId;
```

Comment:
I like seeing an ID on this. You should consider adding IDs to the other models you intend to store in your database.

Reply:
I dislike that the ID is a string. This is a strange choice because IDs are typically numeric.